Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -283,6 +283,68 @@ function MetalSmelter_JobComplete(j)
 	end
 end
 
+function PowerCellPress_UpdateAction(furniture, deltaTime)
+	spawnSpot = furniture.GetSpawnSpotTile()
+	
+	if(spawnSpot.inventory == nil) then
+		if(furniture.JobCount() == 0) then
+			itemsDesired = {Inventory.__new("Steel Plate", 10, 0)}
+			
+			jobSpot = furniture.GetJobSpotTile()
+
+			j = Job.__new(
+			jobSpot,
+			nil,
+			nil,
+			1,
+			itemsDesired,
+			Job.JobPriority.Medium,
+			false
+			)
+			
+			j.RegisterJobCompletedCallback("PowerCellPress_JobComplete")
+			
+			furniture.AddJob(j)
+		end
+	else
+		furniture.ChangeParameter("smelttime", deltaTime)
```

Comment:
@Xeni91 I think you have a small mistake here.
In the Furniture.xml you defined the Params as 
    <Param name="presstime" value="0" />
So it should be presstime not smelttime, or am I reading it wrong?